plugins/redfish/README.md

```diff
@@ -23,8 +23,15 @@ This plugin supports the following protocol ID:
 GUID Generation
 ---------------
 
-These devices use the provided GUID provided in the `SoftwareId` parameter
-without modification. Devices without GUIDs are not supported.
+These devices use the provided GUID provided in the `SoftwareId` property
+without modification if it is a valid GUID. If the property is not a GUID then
+the vendor instance ID is used instead:
+
+* `REDFISH\\VENDOR_${Manufacturer}&SOFTWAREID_${SoftwareId}`
+
+Additionally, this Instance ID is added for quirk and parent matching:
+
+* `REDFISH\VENDOR_${Manufacturer}&ID_${Id}`
```

Comment:
what does `Id` mean in this context?

Reply:
Yes, It certainly wasn't clear to me until you explained it.

Reply:
Okay, fixed, thanks.